### PR TITLE
feature/issue-19-dynamic-cancel-dialog-values

### PR DESCRIPTION
**Pull request by coroutineDispatcher**

### CODE REVIEW

File: example/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="message">Are you sure you want to quit the survey?</string>
+    <string name="title">SurveyKit</string>
+    <string name="yes">Yes</string>
+    <string name="no">No</string>
+</resources>
```
Comment (KlausNie) [inline example/src/main/res/values/strings.xml:7]: Add a new line to the end of the file

File: survey/src/main/java/com.quickbirdstudios.surveykit/backend/views/main_parts/AbortDialogConfiguration.kt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.quickbirdstudios.surveykit.backend.views.main_parts
+
+import androidx.annotation.StringRes
+
+data class AbortDialogConfiguration(
+    @StringRes val title: Int,
+    @StringRes val message: Int,
+    @StringRes val neutralMessage: Int,
+    @StringRes val negativeMessage: Int
+)
```
Comment (KlausNie) [inline survey/src/main/java/com.quickbirdstudios.surveykit/backend/views/main_parts/AbortDialogConfiguration.kt:10]: Add a new line to the end of the file

File: survey/src/main/java/com.quickbirdstudios.surveykit/backend/views/main_parts/DialogConfiguration.kt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package com.quickbirdstudios.surveykit.backend.views.main_parts
+
+data class DialogConfiguration(
+    val title: String,
+    val message: String,
+    val neutralMessage: String,
+    val negativeMessage: String
+)
```
Comment (KlausNie) [inline survey/src/main/java/com.quickbirdstudios.surveykit/backend/views/main_parts/DialogConfiguration.kt:8]: Add a new line to the end of the file

Comment (coroutineDispatcher) [reply survey/src/main/java/com.quickbirdstudios.surveykit/backend/views/main_parts/DialogConfiguration.kt]: ✅

File: survey/src/main/java/com.quickbirdstudios.surveykit/backend/views/main_parts/AbortDialogConfiguration.kt (hunk 2 above)
Comment (coroutineDispatcher) [reply survey/src/main/java/com.quickbirdstudios.surveykit/backend/views/main_parts/AbortDialogConfiguration.kt]: ✅

File: example/src/main/res/values/strings.xml (hunk 1 above)
Comment (coroutineDispatcher) [reply example/src/main/res/values/strings.xml]: ✅

**Pull request merged**